.github/workflows/pending-review-notification.yml

```diff
@@ -22,11 +22,15 @@ jobs:
         with:
           python-version: '3.8.15'
           architecture: 'x64'
-      - uses: DubeySandeep/pending-review-notification@v1
-        with:
-          team-slug: oppia-codeowners
-          repo-token: ${{ secrets.DISCUSSION_NOTIFICATION_TOKEN }}
-          review-turnaround-hours: 48
+      - name: Install dependencies
+        run: pip install -r src_graphql/requirements.txt
+        shell: bash
+      - name: Add path to enviornment
+        run: echo "PYTHONPATH=${{ github.action_path }}" >> $GITHUB_PATH; echo $GITHUB_PATH
+        shell: bash
```

Comment:
Why do we need this? Shouldn't the fact that we use `python -m ...` syntax handle setting up the python path correctly?

Reply:
You are right, we don't need them. Removed!